subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/InstrumentationConfiguration.kt

```diff
@@ -20,11 +20,11 @@ abstract class InstrumentationConfiguration(val name: String) {
     enum class PerformanceType { SIMPLE, MDE }
 
     /**
-     * Если тесты на ветке в упали, сразу после пытаемся прогнать эти упавшие тесты на сборке из таргетной ветки (в которую пытаемся смерджиться)
-     * И если там тоже все плохо - списываем все на инфраструктурные проблемы и позволяем смерджить пулл-реквест
+     * com.avito.performance.PerformanceExtension.targetBranchResultSource
      *
-     * Логика для принятия решения о типе падений в [com.avito.instrumentation.rerun.MergeResultsWithTargetBranchRun]
+     * todo remove in 2020.9
```

Comment:
Lets' add release version to the Deprecated message.
It's convenient to see a version in compilation warnings.